lib/bats-core/test_functions.bash

```diff
@@ -144,6 +148,9 @@ run() { # [--keep-empty-lines] [--output merged|separate|stderr|stdout] [--] <co
     ;;
   esac
 
+  # shellcheck disable=SC2034
+  BATS_TRACE_COMMAND_OUTPUT=("${lines[@]:-}")
```

Comment:
Do we really want to use $lines here ore rather $output?

Reply:
Yep, $output doesn't keep the indentation:

Using $output:

```
❯ TRACE_LEVEL=2 bin/bats --filter "run should exit if tmpdir exist" test/bats.bats
 ✓ run should exit if tmpdir exist
 + bats --tempdir /var/folders/vb/pjyhgj4s491_k0t025_j56y00000gn/T//bats-run-36208-ZnuhSP/BATS_RUN_TMPDIR_TEST.pO5GPI /Users/norsegaud/bats-core/test/fixtures/bats/passing.bats
   Error: BATS_RUN_TMPDIR (/var/folders/vb/pjyhgj4s491_k0t025_j56y00000gn/T//bats-run-36208-ZnuhSP/BATS_RUN_TMPDIR_TEST.pO5GPI) already exists
Reusing old run directories can lead to unexpected results ... aborting!

1 test, 0 failures
```

Using $lines[@]

```
❯ TRACE_LEVEL=2 bin/bats --filter "run should exit if tmpdir exist" test/bats.bats
 ✓ run should exit if tmpdir exist
 + bats --tempdir /var/folders/vb/pjyhgj4s491_k0t025_j56y00000gn/T//bats-run-36294-rsFdVS/BATS_RUN_TMPDIR_TEST.AeY0tI /Users/norsegaud/bats-core/test/fixtures/bats/passing.bats
   Error: BATS_RUN_TMPDIR (/var/folders/vb/pjyhgj4s491_k0t025_j56y00000gn/T//bats-run-36294-rsFdVS/BATS_RUN_TMPDIR_TEST.AeY0tI) already exists
   Reusing old run directories can lead to unexpected results ... aborting!

1 test, 0 failures
```